app/views/demo_mailer/send_worksheets_to_teacher.html.erb

```diff
@@ -19,16 +19,20 @@
                 <% work_group[:users].each do |student| %>
                    <div class="row">
                       <div class="col-sm-2"> <span><div class="circle" style="background-color: <%= student['pen_colour'] %>"></div></span>
-                        <p><%= student['name'] %></p>  
+                        <% if WorkGroup::COLOURS.key(student['pen_colour']) != nil %>
```

Comment:
pen colour should never be nil so this condition isnt necessary in theory

Reply:
Yeah, but I didn't think it would hurt to have? I can get rid of it though

Reply:
lets remove it to keep clean code on the views